tests/foreman/ui/test_hostgroup.py

```diff
@@ -180,16 +296,18 @@ def test_positive_check_activation_keys_autocomplete(self):
 
         @Steps:
 
-        1. Create two content views A & B
-        2. Publish both content views
-        3. Create an activation key pointed to view A in Library
-        4. Create an activation key pointed to view B in Library
-        5. Go to the new Hostgroup page, select content view B & Library,
-            click on the activation key tab and click in the input box
+            1. Create two content views A & B
+            2. Publish both content views
+            3. Create an activation key pointed to view A in Library
+            4. Create an activation key pointed to view B in Library
+            5. Go to the new Hostgroup page, select content view B & Library,
+                click on the activation key tab and click in the input box
```

Comment:
Never thought i'd ever rise such a nitpick, but there's redundant space 😄 There should be 3 of them, not 4 (so 'c' letter is right under 'G').
4 spaces:
![image](https://user-images.githubusercontent.com/11719030/28613124-1cfef13e-71fa-11e7-8ee6-2ae19a6898df.png)

3 spaces:
![image](https://user-images.githubusercontent.com/11719030/28613149-31ea5ff2-71fa-11e7-94c1-0259fdaa2de2.png)

Why did you decide to fix indentation here anyway? Most of our tests with steps just ignore that rule.